scripts/generate-inner.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 #
-for cfile in mit-open-api-clients/config/*.yaml; do
+find . -regex ".*config/.*.yaml" -print0 | while IFS= read -r -d $'\0' cfile; do
```

Comment:
I'm still getting errors because this ends up trying to use yaml files from the learn repo. This change fixes that:

```suggestion
find . -regex "./mit-open-api-clients/config/.*.yaml" -print0 | while IFS= read -r -d $'\0' cfile; do
```

Reply:
I just pushed another variant if you can try it again.